src/cata_tiles.cpp

```diff
@@ -1184,6 +1184,26 @@ void cata_tiles::draw( int destx, int desty, const tripoint &center, int width,
         }
     }
 
+    //Memorize everything the character just saw even if it wasn't displayed.
+    for( int y = 0; y < MAPSIZE * SEEY; y++ ) {
```

Comment:
There is `points_in_radius` function to get collection of tripoints - maybe you can use it here?

Reply:
I would, except that both makes it harder to verify that I'm not re-checking a tile that was just handled, and makes how this code is formatted different from both draw_pixel_minimap and the draw function this is inside (both of which use nested for loops)